db/migrate/20220201134205_create_article_categories.rb

```diff
@@ -0,0 +1,8 @@
+class CreateArticleCategories < ActiveRecord::Migration[6.1]
+  def change
+    create_table :article_categories do |t|
+      t.integer :article_id
+      t.integer :category_id
```

Comment:
Why no created at & updated at?

Reply:
From the UI perspective, there was no particular need of these attributes when it was created.
Is it generally a good practice to include those attributes for each table for traceability in the future if we need to query based on those attributes?

Reply:
Don't tie data to UI.
Data is data. Think about the data as standalone. If you just focus on having the data based on the UI, you will be f-ed when in the future the UI changed and needs created at. Given that you do not track when the data was initially created and updated, that UI change will not be possible. And you will bang head.

So, think in terms of data. Would it make sense to know when this data is created & last updated?

Reply:
Got it. Yep from a data stand point, definitely.